var/spack/repos/builtin/packages/pism/package.py

```diff
@@ -114,30 +117,3 @@ def cmake_args(self):
     def setup_run_environment(self, env):
         env.set("PISM_PREFIX", self.prefix)
         env.set("PISM_BIN", self.prefix.bin)
-
```

Comment:
Removing this as the PETSc version numbers seem not correct making it more confusing than helpful.

Reply:
There seems to be an issue with versioning. If PISM would be tagged 2.1.0 instead of 2.1 it would allow for better tuning of deps. As of now, having 2.1 instead of 2.0.7 in the recipe will be interpreted as 2.1.x which will e.g. fail for dealing with PETSc.

I also bring back the `^petsc~metis~hdf5~hypre~superlu-dist` comment in PISM doc as it now works.

Reply:
> If PISM would be tagged 2.1.0 instead of 2.1 it would allow for better tuning of deps <...>

I'll keep this in mind when tagging future PISM versions -- thanks!